shopfloor/services/cluster_picking.py

```diff
@@ -100,8 +100,11 @@ def _response_for_start_line(self, move_line, message=None, popup=None):
             popup=popup,
         )
 
-    def _response_for_scan_destination(self, move_line, message=None):
-        data = self._data_move_line(move_line)
+    def _response_for_scan_destination(self, move_line, message=None, qty_done=None):
+        if qty_done is None:
+            data = self._data_move_line(move_line)
+        else:
+            data = self._data_move_line(move_line, qty_done=qty_done)
```

Comment:
```suggestion
        data = self._data_move_line(move_line, qty_done=qty_done)
```

Reply:
If the parameter is present in the call the `qty_done` in the data will be changed. We don't want that